pkg/cvo/upgradeable.go

```diff
@@ -291,23 +291,16 @@ func (check *minorVersionClusterUpgradeInProgressUpgradeable) Check() *configv1.
 	}
 
 	currentVersion := clusterversion.GetCurrentVersion(cv.Status.History)
-	// This can occur in early start up when the configmap is first added and version history
+	klog.V(2).Infof("The current version is %s and the desired version is %s", currentVersion, cv.Status.Desired.Version)
+	// currentVersion == "" can occur in early start up when the configmap is first added and version history
 	// has not yet been populated.
-	if currentVersion == "" {
-		klog.V(2).Infof("current version is empty")
-		return nil
-	}
-
-	currentMinor := clusterversion.GetEffectiveMinor(currentVersion)
-	desiredMinor := clusterversion.GetEffectiveMinor(cv.Status.Desired.Version)
-	klog.V(2).Infof("The current minor version is %s and the desired minor version is %s", currentMinor, desiredMinor)
-	if !clusterversion.MinorVersionUpgrade(currentMinor, desiredMinor) {
+	if currentVersion == "" || cv.Status.Desired.Version == "" || currentVersion == cv.Status.Desired.Version {
 		return nil
 	}
 
-	message := fmt.Sprintf("There is a minor level upgrade from %s to %s in progress.", currentVersion, cv.Status.Desired.Version)
+	message := fmt.Sprintf("There is an upgrade from %s to %s in progress.", currentVersion, cv.Status.Desired.Version)
```

Comment:
possibly a more robust way to implement this guard would be to ignore `history` and just echo the `Progressing` status (which is also `True` during installs, and blocking folks from heading to 4.(y+1) before their install completes seems reasonable).

Reply:
https://github.com/openshift/cluster-version-operator/pull/1080/commits/74d4e0437723ab4bf3e2c46cf20b8e9a2f78b11e to address this.

It seems lots of simpler than before.